lib/node_modules/@stdlib/math/base/special/frexp/docs/types/index.d.ts

```diff
@@ -23,30 +23,75 @@
 import { Collection } from '@stdlib/types/object';
 
 /**
-* Splits a double-precision floating-point number into a normalized fraction and an integer power of two.
-*
-* ## Notes
-*
-* -   The first element of the returned array is the normalized fraction and the second is the exponent. The normalized fraction and exponent satisfy the relation `x = frac * 2^exp`.
-* -   If provided positive or negative zero, `NaN`, or positive or negative infinity, the function returns a two-element array containing the input value and an exponent equal to zero.
-* -   For all other numeric input values, the absolute value of the normalized fraction resides on the interval [0.5,1).
-*
-* @param out - output array
-* @param x - input value
-* @returns output array
-*
-* @example
-* var Float64Array = require( `@stdlib/array/float64` );
-*
-* var out = new Float64Array( 2 );
-*
-* var y = frexp( out, 4.0 );
-* // returns <Float64Array>[ 0.5, 3 ]
-*
-* var bool = ( y === out );
-* // returns true
-*/
-declare function frexp( out: Collection, x: number ): Collection;
+ * Inteface describing `frexp`.
+ */
+interface Frexp {
+	/**
+	* Splits a double-precision floating-point number into a normalized fraction and an integer power of two.
+	*
+	* ## Notes
+	*
+	* -   The first element of the returned array is the normalized fraction and the second is the exponent. The normalized fraction and exponent satisfy the relation `x = frac * 2^exp`.
+	* -   If provided positive or negative zero, `NaN`, or positive or negative infinity, the function returns a two-element array containing the input value and an exponent equal to zero.
+	* -   For all other numeric input values, the absolute value of the normalized fraction resides on the interval [0.5,1).
+	*
+	* @param x - input value
+	* @returns output array
+	*
+	* @example
+	* var out = frexp( 4.0 );
+	* // returns [ 0.5, 3 ]
+	*
+	* @example
+	* var out = frexp( 0.0 );
+	* // returns [ 0.0, 0 ]
+	*
+	* @example
+	* var out = frexp( -0.0 );
+	* // returns [ -0.0, 0 ]
+	*
+	* @example
+	* var out = frexp( NaN );
+	* // returns [ NaN, 0 ]
+	*
+	* @example
+	* var out = frexp( Infinity );
+	* // returns [ Infinity , 0 ]
+	*
+	* @example
+	* var out = frexp( -Infinity );
+	* // returns [ -Infinity , 0 ]
+	*/
+	( x: number ): Collection;
```

Comment:
```suggestion
	( x: number ): Array<number>;
```

For the main export, only an `Array` containing numbers is returned.